packages/plugins/typed-modal/.eslintrc.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Can you move this up to `packages/plugins/` so that you don't have to recreate it for each plugin?

Reply:
done